website/docs/core-concepts/writing-code/javascript-editor-beta/README.md

```diff
@@ -151,28 +151,35 @@ You’ll see a Snippets button available on the right top side of the editor. Cl
 
 ![Snippets Button](</img/JavaScript_Editor__Snippets.png>)
 
-#### Appsmith Snippet Library
+#### Appsmith snippet library
 
 You can search and copy the Snippets from the Appsmith Snippet Library and use them in the JavaScript Editor. You can then build on the code snippet to add your code or use it as is.
 
-![Appsmith Snippet Library](</img/JavaScript_Editor__Snippets__Appsmith_Snippet_Library.png>)
+![Appsmith snippet library](</img/JavaScript_Editor__Snippets__Appsmith_Snippet_Library.png>)
 
 :::info
-Appsmith is actively working on expanding the **Snippets Library**. Please reach out to us on[ Discord](https://discord.com/invite/rBTTVJp) or[ Community Forms](https://community.appsmith.com/) if you wish to contribute to the library.
+Appsmith is actively working on expanding the **Snippets Library**. Please reach out on[ Discord](https://discord.com/invite/rBTTVJp) or[ Community Forms](https://community.appsmith.com/) if you wish to contribute to the library.
 :::
 
-### Debugger Statements
+### Debugger statements
```

Comment:
Actually I kept the debugger statement as is because it is under the section working with JavaScript Editor that actually talks about the available features of the editor. And so added the two subsections for debugging under it. If we need to modify it we can do as part of Rehaul for the JS Objects. For now, I kept it as is so that the flow of the doc is unchanged.

Reply:
Hmm, I still think this deserves a separate sub-section because we now have enough info on debugging errors for this page. Logically the flow looks wrong at the moment. 
<img width="806" alt="Screenshot 2022-12-12 at 18 39 09" src="https://user-images.githubusercontent.com/32227135/207053168-bdf4e7e2-1ed4-49a3-ac04-d18ed56db0b4.png">

You can still keep it under 'Working with JavaScript Editor' but with the right structure

Reply:
Let's discuss this.